test/ServerComparison.FunctionalTests/ResponseTests.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace ServerComparison.FunctionalTests
 {
-    // Uses ports ranging 5070 - 5079.
+    // Uses ports ranging 5070 - 5089.
```

Comment:
Rather than doing this, we hsould have a small tool that checks that each test got a unique port (using reflection)